Plugins/InstallSafeDITool/InstallCLIPluginCommand.swift

```diff
@@ -0,0 +1,109 @@
+// Distributed under the MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import Foundation
+import PackagePlugin
+
+@main
+struct InstallSafeDITool: CommandPlugin {
+    func performCommand(
+        context: PackagePlugin.PluginContext,
+        arguments _: [String]
+    ) async throws {
+        guard let safeDIOrigin = context.package.dependencies.first(where: { $0.package.displayName == "SafeDI" })?.package.origin else {
+            Diagnostics.error("No package origin found for SafeDI package")
+            return
+        }
+        switch safeDIOrigin {
+        case let .repository(url, displayVersion, _):
+            // As of Xcode 16.0 Beta 6, the display version is of the form "Optional(version)".
```

Comment:
Apple... why...

Using `displayVersion` is clearly brittle, but it does work.

Reply:
Watch them add a space

Reply:
Seemed like an oversight. We'll see https://github.com/swiftlang/swift-package-manager/pull/7934

Reply:
If this ever fully breaks we can do what Apollo does and [hardcode the current release version](https://github.com/apollographql/apollo-ios/blob/f0c89da06e252c91920fe1e26956b14ba1b17d0a/Sources/Apollo/Constants.swift#L4). I'd prefer to trust SPM rather than have to keep this in-sync myself, so we're starting out with this approach.